schemas/snuba-lw-deletions.v1.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "lw_delete",
+  "type": "object",
+  "properties": {
+    "storage_name": {
+      "type": "string"
+    },
+    "rows_to_delete": {
+      "type": "integer"
+    },
+    "conditions": {
+      "type": "object",
+      "oneOf": [{ "$ref": "#/definitions/SearchIssuesConditions" }]
+    },
+    "tenant_ids": {
+      "type": "object",
+      "additionalProperties": {
+        "anyOf": [{ "type": "string" }, { "type": "integer", "minimum": 1 }]
+      }
+    }
+  },
+  "required": ["storage_name", "rows_to_delete", "conditions"],
+  "definitions": {
+    "SearchIssuesConditions": {
+      "title": "search_issues_conditions",
+      "type": "object",
+      "properties": {
+        "project_id": {
+          "type": "array",
+          "items": { "type": "integer" }
+        },
+        "group_id": {
+          "type": "array",
+          "items": { "type": "integer" }
+        }
+      },
+      "additionalProperties": false
+    }
+  }
+}
```

Comment:
this now means that `project_id` and `group_id` being required for a delete is enforce (and would need to be changed) in at least 2 places now instead of 1. at the api schema and the topic definition.

i think its fine, but it makes me think sql query might be better for the topic than this object.

Reply:
I think its good to have the validation in the kafka topic, in case we ever make the change to produce messages to the topic from a different place than the api we have. Additionally, the validation we have in the API is to make sure that we can respond to the user if they have the wrong column/column type etc, whereas this validation is to make sure our logic doesn't make a mistake when producing messages, and that error would be seen in the consumer not the api. So I think both are needed